requirements.txt

```diff
@@ -1,15 +1,15 @@
+# This file is also processed by conda-build, so keep spaces as "pkg >=1.2"
+appdirs
+dask >=0.17.0
+holoviews
+ipywidgets >=7.2
 jinja2
 msgpack-numpy
 msgpack-python
 numpy
 pandas
-pytest
```

Comment:
I suppose there could be a test-requires.txt, but this is already covered in meta.yaml.

Reply:
Yes at least for Bokeh we only really support test/development from conda-based envs. If you want to be more broad than that, a `test-requires.txt` is an option, or else it could just be made a runtime dependency all the time.

Reply:
Let's leave for now. Documentation on dev/test should be written at some point.

Reply:
Just to make sure I understand, leave things as they currently are, without `pytest` ?

Reply:
remove pytest

Reply:
removed in https://github.com/ContinuumIO/intake/pull/211/commits/6fcc5a8dc225d7d7cc6de4a11ed2798476e84f1b